charts/helm-dep-installer/values.yaml

```diff
@@ -0,0 +1,36 @@
+---
+# -- Image used to for installer/delete job
+image: ghcr.io/dtzar/helm-kubectl:3.9.4  # TODO: Rerelease with more official repo (eg ghcr.io/caraml-dev)?
+
+helmChart:
+  # -- repository of helm chart to install
+  repository: "https://charts.bitnami.com/bitnami"
+  # -- Chart name
+  chart: "nginx"
```

Comment:
assuming these values are just for default testing of this chart, am I right?

Reply:
Yes, it's to allow the chart testing to pass and provide an example on how to configure the values